### PR TITLE
rgw/v2/tests/s3_swift: user_create.py 

### PR DESCRIPTION
**Pull request by viduship**
User_create.py file to create one tenanted and non tenanted user.

Logs : http://magna002.ceph.redhat.com/ceph-qe-logs/vidushi/automation/PRs/98_ceph-qe_scripts/

Signed-off-by: viduship <vimishra@redhat.com>

### CODE REVIEW

File: rgw/v2/tests/s3_swift/user_create.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    test_info = AddTestInfo('create users')
+    io_info_initialize = IOInfoInitialize()
+    basic_io_structure = BasicIOInfoStructure()
+    io_info_initialize.initialize(basic_io_structure.initial())
+    try:
+        test_info.started_info()
+        # create a non-tenanted user
+        if config.user_type == 'non-tenanted':
+            all_users_info = s3lib.create_users(config.user_count, config.cluster_name)
+            with open('../../lib/user_details.json', 'w') as fout:
```
Comment (rakeshgm) [inline rgw/v2/tests/s3_swift/user_create.py:35]: take the abs path and open the file.

File: rgw/v2/tests/s3_swift/user_create.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+                json.dump(all_users_info, fout)
+            test_info.success_status('non-tenanted users creation completed')
+        else:
+            log.info('create tenanted users')
+            for i in range(config.user_count):
+                tenant_name = 'tenant' + str(i)
+                all_users_info = s3lib.create_tenant_users(config.user_count, tenant_name, config.cluster_name)
+                with open('../../lib/user_details.json', 'w') as fout:
```
Comment (rakeshgm) [inline rgw/v2/tests/s3_swift/user_create.py:43]: take the abs path and open the file.

File: rgw/v2/tests/s3_swift/user_create.py (hunk 1 above)
Comment (viduship) [reply rgw/v2/tests/s3_swift/user_create.py]: done.

File: rgw/v2/tests/s3_swift/user_create.py (hunk 2 above)
Comment (viduship) [reply rgw/v2/tests/s3_swift/user_create.py]: done

**Review by rakeshgm — APPROVED** [review]

**Pull request merged**